tests/test-helpers.js

```diff
@@ -0,0 +1,43 @@
+const nixt = require("nixt");
+const fs = require("fs");
+const path = require("path");
+const mkdirp = require("mkdirp");
+
+exports.createFixtureIfDoesntExist = function (name, args) {
```

Comment:
In these tests, a fixture is a directory with a full project in it generated by create-single-spa. It takes a while to `yarn install`, so for local testing I'm not destroying/recreating every time so that tests aren't awful.